sigs.yaml

```diff
@@ -492,23 +492,30 @@ sigs:
       establishment of conventions for writing CLI commands, POSIX compliance,
       and improving the command line tools from a developer and devops user
       experience and usability perspective.
-    charter_link:
+    charter_link: charter.md
     label: cli
     leadership:
       chairs:
+      - name: Maciej Szulik
+        github: soltysh
+        company: Red Hat
+      - name: Sean Sullivan
```

Comment:
seems like it might be better to make chair changes separately from the charter addition

Reply:
I'd just move this to separate commit, which should be sufficient.